### PR TITLE
[IMPROVE] Rewrite Omnichannel Queue Page to React

### PR DESCRIPTION
**Pull request by tiagoevanp**
## Proposed changes (including videos or screenshots)

image

### CODE REVIEW

**Comment by KevLehman** [comment]
@tiagoevanp can you fix the conflicts on this one? 🙏

**Comment by tiagoevanp** [comment]
> @tiagoevanp , is intended that the old `livechat-queue` route is still accessible? (though showing undefined instead of rendering the template) image
> 
> Also, the icon on the sidebar is not appearing on this branch :( image

Have you adjusted the config to activate the queue list?

**Comment by KevLehman** [comment]
Yup, it was active, but for some reason clicking on the button was just sending me to the old page. Now i can see it without issues, so would be a matter of updating the react version to be the "main" one

File: client/startup/routes.tsx [hunk 1]
```diff
@@ -136,6 +136,16 @@ FlowRouter.route('/omnichannel-directory/:page?/:bar?/:id?/:tab?/:context?', {
 	},
 });
 
+FlowRouter.route('/queue-list', {
+	name: 'queue-list',
+	action: () => {
+		const OmnichannelQueueList = createTemplateForComponent('QueueList', () => import('../views/omnichannel/queueList'), {
+			attachment: 'at-parent',
+		});
+		appLayout.render('main', { center: OmnichannelQueueList });
```
Comment (KevLehman) [inline client/startup/routes.tsx:145]: Shouldn't this be `renderMainLayout`?

**Review by KevLehman — APPROVED** [review]

**Pull request merged**